frontend/min-app/src/components/MineralTable.tsx

```diff
@@ -18,18 +18,29 @@ const MineralTable = () => {
 		setMinerals(mineralList);
 	};
 
+
 	return (
 		<div>
 			<table>
 				<tr>
-					<th>Which rock</th>
-					<th>Found in</th>
+					<th>Classificazione</th>
+					<th>Minerale</th>
+					<th>Trovato in</th>
 				</tr>
-				{minerals.map(min => {
+				{/* {minerals.map(min => {
 					return (
 						<TableRow key={min.item_id} mineral={min} />
 					);
-				})};
+				})}; */}
+				{minerals.map((val, key) => {
+                    return (
+                        <tr key={key}>
```

Comment:
Mikä toi key on ja mistä se tulee?

Reply:
Se oli siinä aiemmin, mutta voin ottaa pois

Reply:
Eikun pitää olla, muuten `Line 30:25:  Missing "key" prop for element in iterator  react/jsx-key`